fe/fe-core/src/main/java/com/starrocks/analysis/CreateRoutineLoadStmt.java

```diff
@@ -114,6 +119,15 @@ public class CreateRoutineLoadStmt extends DdlStmt {
     public static final String KAFKA_OFFSETS_PROPERTY = "kafka_offsets";
     public static final String KAFKA_DEFAULT_OFFSETS = "kafka_default_offsets";
 
+    // pulsar type properties
+    public static final String PULSAR_SERVICE_URL_PROPERTY = "pulsar_service_url";
+    public static final String PULSAR_TOPIC_PROPERTY = "pulsar_topic";
+    public static final String PULSAR_SUBSCRIPTION_PROPERTY = "pulsar_subscription";
+    // optional
+    public static final String PULSAR_PARTITIONS_PROPERTY = "pulsar_partitions";
+    public static final String PULSAR_INITIAL_POSITIONS_PROPERTY = "pulsar_initial_positions";
+    public static final String PULSAR_DEFAULT_INITIAL_POSITION = "pulsar_default_initial_position";
```

Comment:
What's the difference between `pulsar_initial_positions` and `pulsar_default_initial_position` ? It seems that they can only be the same enum vallue.

Reply:
`pulsar_default_initial_position` will be valid that when `pulsar_initial_positions` is not set, if user don't want to specify same position for each partition.